### ISSUE TITLE
Translate German to English

### ISSUE THREAD

**Issue by maarten-pennings** [harbaum/upide#3]
The examples have german name, rest of the app is English.

**Comment by harbaum** [comment]
These examples are for a german Micropython book I am writing.

Actually "the rest of the app" is not just english but switches to german on a german operating system. But that doesn't work with the examples. I was already thinking about making them bi-lingual. But parts of the code also shows up inside the book and being be-lingual there would be a waste of paper.

But I could implement something that points to a complete different example if it finds a translated example online. I have to think about that.

**Comment by maarten-pennings** [comment]
Hmm understand the problem.
Maybe you can make a DE and EN directory in the examples...

**Comment by harbaum** [comment]
Yeah. I could even load the right one automatically. I'll have a look at that.
